packages/holocron-dev-server/README.md

```diff
@@ -0,0 +1,166 @@
+[one-app-bundler]: https://github.com/americanexpress/one-app-cli/tree/main/packages/one-app-bundler
+[dll-plugin]: https://webpack.js.org/plugins/dll-plugin/
+[providedExternals & requiredExternals]: https://github.com/americanexpress/one-app-cli/tree/main/packages/one-app-bundler#providedexternals--requiredexternals
+[one-app-runner]: https://github.com/americanexpress/one-app-cli/tree/main/packages/one-app-runner
+[express]: https://github.com/expressjs/express
+[memfs]: https://github.com/streamich/memfs
+[webpack]: https://github.com/webpack/webpack
+[webpack-dev-middleware]: https://github.com/webpack/webpack-dev-middleware
+[webpack-hot-middleware]: https://github.com/webpack-contrib/webpack-hot-middleware
+[webpack bundle analyzer]: https://github.com/webpack-contrib/webpack-bundle-analyzer
+[react-refresh-webpack-plugin]: https://github.com/pmmmwh/react-refresh-webpack-plugin
+[react-refresh]: https://github.com/facebook/react/tree/master/packages/react-refresh
+[react-refresh-troubleshooting]: https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/main/docs/TROUBLESHOOTING.md
+[docker]: https://www.docker.com/
+[React Dev tools extension]: https://reactjs.org/blog/2019/08/15/new-react-devtools.html
+[esbuild-loader]: https://github.com/privatenumber/esbuild-loader
+# @americanexpress/holocron-dev-server
+
+`@americanexpress/holocron-dev-server` is an **experimental** development server
+made for [One App][One APP] Holocron modules designed for enabling fast refresh and reloading of assets
+while creating web experiences.
+
+If you are looking for a **production-like environment**
+to run One App Holocron modules, [`@americanexpress/one-app-runner`][one-app-runner] would be the right tool of choice.
+
+## 📖 Table of Contents
+
+* [Features](#-features)
+* [Usage](#-usage)
+* [API](#%EF%B8%8F-api)
+* [Troubleshooting](#-troubleshooting)
+
+## ✨ Features
+
+- Runs One App Holocron modules on an experimental development server
+- [Fast refresh][react-refresh] for Holocron modules, React components and styles are all hot loaded in the browser
+- Zero-config start up with pre-existing `"one-amex"` configuration in the `package.json`
+- Hot reload [parrot][parrot] scenarios and watch for changes
+- Hot reload language packs and watch for changes from the local modules
+- Combines the remote module if provided with the local module map
+ remote module map with the universal map
+- [providedExternals & requiredExternals][providedExternals & requiredExternals] are pre-built with [esbuild-loader] as a [DLL][dll-plugin] bundle to speed up build times of the local modules
+- local Holocron modules bundles analyzed and reported using [webpack bundle analyzer][webpack bundle analyzer]
+
+### Caveats
+- No support for custom `babel` config or auto-merge option (with zero-config)
+
+## 🤹‍ Usage
+
+> **Prerequisites**
+> - [Docker][docker] is required to load One App statics from development environment.
```

Comment:
We could download the one app statics from the CDN without the need for docker and one app runner i.e. https://cdn.jsdelivr.net/npm/@americanexpress/one-app-statics@5.10.1/
or the ZIP file from the Github release.


This is probably a future enhancement not a breaking issue right now